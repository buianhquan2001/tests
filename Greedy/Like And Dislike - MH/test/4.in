5
37 22 30
63 98 28
1 19 15
37 0 9
98 69 100
