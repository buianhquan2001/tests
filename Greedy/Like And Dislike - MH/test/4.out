Unknown
Dislike
Dislike
Like
Unknown
